### PR TITLE
Update sha256.md - add virustotal

### PR DESCRIPTION
**Pull request by commitay**
Add VirusTotal codesigning information.

Closes https://github.com/caskroom/homebrew-cask/issues/34554

/CC @vitorgalvao

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Pinging @caskroom/maintainers for comments.

Note that we still need to pay attention to the signer. After all, a malicious actor could sign their malicious change (as happened to Transmission, a while back).

**Review by vitorgalvao — APPROVED** [review]

**Review by victorpopkov — APPROVED** [review]

File: doc/cask_language_reference/stanzas/sha256.md [hunk 1]
```diff
@@ -18,10 +18,19 @@ We use a checksum whenever possible.
...
+   - If confirmed on GitHub, @mention [one of the Homebrew-Cask maintainers](https://github.com/orgs/caskroom/people).
+   - Otherwise, post a link to the developer's confirmation.
+ - By codesign:
+   - If the Cask is an `.app` that is [codesigned](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/codesign.1.html) (in a `.dmg` or `.zip` container) it can be uploaded and verified using [VirusTotal](https://www.virustotal.com/) by looking at the “File Details” tab. 
+   
+     Here's an example for [Brave-0.15.2.dmg](https://www.virustotal.com/en/file/56ef38654223c910dd8219fce426e27b613eedb900608304c260021f7f774210/analysis/):
+   
+     <img src="https://i.imgur.com/MTuGLPy.png" width="653px" alt="VirusTotal codesigning">
+
+Maintainers will confirm the VirusTotal submission is legitimate by comparing it’s `sha256` with the one on the updated cask.
```
Comment (giannitm) [inline doc/cask_language_reference/stanzas/sha256.md]: Should be "its" not "it's".

Comment (giannitm) [inline doc/cask_language_reference/stanzas/sha256.md]: Should be "its" not "it's".

**Review by gtmgianni — CHANGES_REQUESTED** [review]

**Review by miccal — APPROVED** [review]

File: doc/cask_language_reference/stanzas/sha256.md [hunk 2]
```diff
@@ -33,4 +33,4 @@ The confirmation of the updated `sha256` should ideally be publicly available. S
    
      <img src="https://i.imgur.com/MTuGLPy.png" width="653px" alt="VirusTotal codesigning">
 
-Maintainers will confirm the VirusTotal submission is legitimate by comparing it’s `sha256` with the one on the updated cask.
+Maintainers will confirm the VirusTotal submission is legitimate by comparing its `sha256` with the one on the updated cask.
```
Comment (commitay) [inline doc/cask_language_reference/stanzas/sha256.md]: Should this be moved above the example?

Comment (miccal) [inline doc/cask_language_reference/stanzas/sha256.md]: I think it is fine where it is.

**Pull request merged**